packages/react/src/components/Modal/Modal.tsx

```diff
@@ -92,7 +92,7 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>(function Modal(
             color='neutral'
             icon
             name='close'
-            size='md'
+            data-size='md'
```

Comment:
Suggest removing this default `data-size` as it ignores data-size on `Modal`.

Reply:
This is for the button inside a modal

Reply:
but this should scale when you set `data-size`, so I'll remove it